tools/Iwyu/iwyu.imp

```diff
@@ -24,7 +24,7 @@
   { symbol: ["PUP::er", private, "<pup.h>", public]},
 
   { include: ["<catch.hpp>", public,
-              "\"tests/Unit/TestingFramework.hpp\"", public]},
+              "\"Framework/TestingFramework.hpp\"", public]},
```

Comment:
@fmahebert You've been playing with IWYU.  Maybe this is interesting to you.

Reply:
Thanks for catching this! This may require conflict resolution between our PRs but if so it'll be easy.